.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Please add  `test.md` to the .gitignore file it is part of requierement. :smile: